homeassistant/components/rpi_power/binary_sensor.py

```diff
@@ -31,43 +34,22 @@ async def async_setup_entry(
 class RaspberryChargerBinarySensor(BinarySensorEntity):
     """Binary sensor representing the rpi power status."""
 
-    def __init__(self, under_voltage):
+    def __init__(self, under_voltage: UnderVoltage) -> None:
         """Initialize the binary sensor."""
         self._under_voltage = under_voltage
-        self._is_on = None
+        self._attr_is_on = None
```

Comment:
This is the default already, no need to set that again.

```suggestion
```

Reply:
done